.github/workflows/test_pipeline.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           python -m pip install requests
           python -m pip install datetime
           python -m pip install pypistats
```

Comment:
those need to go into `requirements.txt`  and installed with `... -r requirements.txt`

Reply:
Will do it after the merge